Demo/AppDelegate.swift

```diff
@@ -32,6 +32,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         window?.tintColor = .systemBlue
         window?.backgroundColor = Theme.UITableViewController.backgroundColor
 //        ThemeManager.applyWhiteTheme()
+        
```

Comment:
Why this change?

Reply:
To start travis CI build again you don't need to commit anything, just do a push from your branch and it will trigger travis ci build